src/components/AppCard/styles.js

```diff
@@ -22,6 +22,7 @@ const cardActions = css`
 	position: relative;
 	overflow: hidden;
 	padding: 25px auto;
+	max-height: 205px;
```

Comment:
This will break the responsive view. Please do thorough testing before committing any changes.

Reply:
Fixed it was only breaking in screen less than 400px